js/app/location.js

```diff
@@ -47,8 +48,8 @@ define([
                 }
             },
             draw: (renderer, color) => {
-                var topLeftCorner = renderer.convertCoordonateToBackground(data.description.topLeftCorner);
-                var bottomRightCorner = renderer.convertCoordonateToBackground(data.description.bottomRightCorner);
+                var topLeftCorner = renderer.convertCoordonateToBackground(data.description.topLeftCorner, true);
+                var bottomRightCorner = renderer.convertCoordonateToBackground(data.description.bottomRightCorner, true);
                 var canvasContext = renderer.getContext();
```

Comment:
what are all the "true" for ?

Reply:
For displaying: When we use the number for display, they all have to be doubled on retina display, which is different when using the number for mouseover.

Reply:
hmm ok, but i didnt see this argument being added to the methods. did I miss it ?